app.py

```diff
@@ -1,2 +1,40 @@
-from flask import Flask, request
+from flask import Flask, request, jsonify
 import json
+from flask.views import MethodView
+
+
+app = Flask(__name__)
+with open("restaurants.json", "r") as read_file:
+    data = json.load(read_file)
+
+restaurants = data['restaurants']
+
+
+class RestaurantEndpoint(MethodView):
+    def get(self):
+        return jsonify({'restaurants': restaurants}), 200
+
+    def post(self):
+        content = request.json
+        if content is not None:
```

Comment:
the point of `POST` method is to create a new record. if the body is empty (`if content is None`), then it just does not make sense, it is wrong input. so it would be more logical to raise something like Flask's `ValidationError` if body is empty

Reply:
fixed